manual/dotnet4.7.2-deu/dotnet4.7.2-deu.nuspec

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+  <metadata>
+    <id>dotnet4.7.2-deu</id>
+    <title>Microsoft .NET Framework 4.7.2 German Language Pack</title>
+    <version>4.6.1590.0</version>
```

Comment:
Shouldn't the version be `4.7.3062.0`?

Reply:
yes, 
4.6.1590.0 |  .NET 4.6.2 (released on 20th July 2016)

Reply:
Fixed